### DISCUSSION TITLE
실제 플레이어가 소유한 캐릭터가 아닌 제페토 캐릭터 NPC를 월드에 불러올 수 있나요?

### DISCUSSION THREAD

**Discussion by aloha-connor** [naverz/zepeto-studio-kor#1280] [5. Q&A]
`CreateByZepetoId(zepetoId: string, spawnInfo: SpawnInfo, complete: System.Action$1<ZepetoCharacter>)`를 사용해서 zepetoId를 사용해 해당 Id와 같은 모습의 NPC를 만들 수 있다는 건 알고 있습니다.

그런데 실제 플레이어가 아닌, 제페토 캐릭터를 커스터마이즈한 NPC를 월드에 집어넣고 싶은데요, 혹시 가능할까요? (빌드잇에서 제공하는 Zombie나 Bildy 같이...)
아니면 NPC는 반드시 외부에서 가져온 모델링을 써야할까요?

**Comment by zlmrzlo** [comment]


**Reply by aloha-connor** [reply]
이~ 그러면 머리나 의상 같은 건 에셋을 따로 구해서 적용해야하는 건가요?

**Reply by aloha-connor** [reply]
제페토에서 제공하는 아이템들로 코디한 NPC를 만들 수 있냐는 질문이었습니다! 아무래도 안되는 거 같군요 ㅠ

**Reply by zlmrzlo** [reply]
질문하신 내용을 보니
제페토 캐릭터를 커스터마이즈한 것을 가지고 오고 싶은데
지금 알고 있는 지식으로는 내가 플레이하는 캐릭터만 가지고 올 수 있는 정도라는 것인가요?

**Reply by zlmrzlo** [reply]
저는 외부에서 커스터마이즈하고 가지고 오려고 하는 것으로 잘못 이해했네요.
제페토 캐릭터를 커스터마이즈하고 월드로 가지고 와서 NPC 처럼 사용 가능해요.

**Reply by zlmrzlo** [reply]
https://studio.zepeto.me/guides/zepetocharacter
위의 링크가 관련 내용이 들어간 가이드에요.
질문하셨던 부분은 이미 가이드에 있던 내용이라
다른 말을 하는 줄 알고 좀 어려운 방식으로 알려드린 것 같네요.

**Reply by aloha-connor** [reply]
음 설명드리기가 좀 어렵네요..
그러니까 저 방법을 사용하면 실제 플레이어의 캐릭터를 불러올 수 있는 것까지는 가능한데, 일종의 가상 플레이어를 만들어서 캐릭터를 만들고 그 캐릭터를 불러올 수 있는지가 궁금했습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 저희가 9월 업데이트 내용으로 마네킹 기능을 준비하고 있습니다.
다음과 같은 내용이 가능해질 예정입니다.
- 더미 NPC를 생성한 후 아이템 ID를 입력해서 원하는 의상을 장착해서 마네킹을 만들수 있음
- 마네킹 또는 특정 오브젝트와 상호작용해서 의상을 입어보기 할 수 있음
- 마네킹 또는 특정 오브젝트와 상호작용해서 특정 의상을 구매할 수 있음 (단, 내가 제작한 아이템에 한함)

업데이트 노트에서 상세한 가이드와 함께 제공될 예정이오니 조금만 기다려 주세요!
감사합니다.

**Reply by aloha-connor** [reply]
감사합니다!
